dotcom-rendering/src/web/components/ArticleTitle.tsx

```diff
@@ -65,30 +77,43 @@ export const ArticleTitle = ({
 	guardianBaseURL,
 	badge,
 	isMatch,
-}: Props) => (
-	<div css={[sectionStyles, badge && badgeContainer]}>
-		{badge && format.display !== ArticleDisplay.Immersive && (
-			<div css={titleBadgeWrapper}>
-				<Badge imageUrl={badge.imageUrl} seriesTag={badge.seriesTag} />
+}: Props) => {
+	const palette = decidePalette(format);
+	return (
+		<div css={[sectionStyles, badge && badgeContainer]}>
+			{format.design === ArticleDesign.LiveBlog && (
+				<span css={livePulseIconStyles(palette)}>
```

Comment:
See earlier comment about these being inside the `PulsingDot` component. Also does it want to be a span or is it better as a div?

Reply:
Sorry, I made some changes and the `PulsingDot` is now being added from within the `SeriesSectionLink`. I removed the span in there :)

Reply:
I had to again use a span around `Island` in `SeriesSectionLink`. Do you still think a `div` could be more suitable rather than a `span`?

Reply:
I think a `span` probably makes sense here. The reason I flagged it was because often spans are used just to wrap text, here we're wrapping an Island and a component, though essentially that component is itself just another `span`.

If you wanted to use a `div` just add a `display: inline` to the div styles, I'm not sure how much it matters though really.